x-element.js

```diff
@@ -686,6 +685,18 @@ export default class XElement extends HTMLElement {
   }
 
   // Called once per-host from initial "connectedCallback".
+  static #connectHost(host) {
+    const initialized = XElement.#initializeHost(host);
+    XElement.#addListeners(host);
```

Comment:
Again, this is the magic-foo here. See how we split apart _initialization_ and _initial render_? This allows us to sneak in _adding event listeners_ between these two points in time.